calcom/cal.com
# When deleting video app, change locations to Cal Video

## What does this PR do?



When deleting a video app, any event types that use that app will now revert to Cal video. Any previously created URLs with that app will still be valid.

Part of #2965

[](https://www.loom.com/share/f34de74a631f49508139e8d9e2060beb)

**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Install a video app (Not Cal video or Google Meet)
- Go to an event type and add that installed video app as a location
- Go to installed apps and delete the video app
- Go back to the edited event type. The location should have changed to Cal video

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
